README.md

```diff
@@ -103,6 +103,30 @@ find(
 ): Promise<PagedResult<Schema>>;
 ```
 
+## 8.0.0 Upgrade Guide
+
+### Double encoding RQL values 
+From version 8.0.0 RQL values are now [double encoded](https://docs.extrahorizon.com/extrahorizon/additional-resources/resource-query-language-rql#double-encoding-of-special-characters) by default when using the rql builder, to support the use of special characters in rql operations.
+
+In your existing application it is likely that you have encoded values provided to the rql builder using methods such as `encodeURIComponent()`. To support the automatic double encoding of special characters, where applicable, these instances of encoding should be removed.
+- Before 8.0.0: `rqlBuilder().eq('diagnosis', encodeURIComponent('Hypertension - STAGE 1'))`
+- After 8.0.0: `rqlBuilder().eq('diagnosis', 'Hypertension - STAGE 1')`
+
+Any use of an rql builder that does not need to support special characters may remain as it is.
```

Comment:
What is the purpose of this sentence? 🤔 
I don't think it says a lot.

Reply:
We also have to inform users that they only need to make changes in the scenarios that they are using special characters no?

e.g. `rqlBuilder().eq('diagnosis', 'Hypertension')` would require no changes.